tests/TestCase/Model/EndpointTest.php

```diff
@@ -358,4 +358,18 @@ public function testSchema()
             $endpoint->schema()
         );
     }
+
+    public function testFindWithSelectAndWhere()
```

Comment:
Multiple methods should not be tested inside a single test method :)

Reply:
I just figured it would be nice to have a more integration focused test, as this is how I'd imagine it being used in the wild.

Perhaps it could be moved into a Integration test class? Happy to remove it if you feel that there is enough coverage of the methods used.

Reply:
Nah it's okay. As long as there is decent test coverage I don't want to be pedantic about it.